pkg/controller/performanceprofile/components/tuned/tuned_test.go

```diff
@@ -173,7 +173,7 @@ var _ = Describe("Tuned", func() {
 					reservedSet, err := cpuset.Parse(string(*profile.Spec.CPU.Reserved))
 					Expect(err).ToNot(HaveOccurred())
 					reserveCPUcount := reservedSet.Size()
-					channelsRegex := regexp.MustCompile(`\s*channels=combined ` + strconv.Itoa(reserveCPUcount) + `\s*`)
+					channelsRegex := regexp.MustCompile(`\s*channels=combined\s*` + strconv.Itoa(reserveCPUcount) + `\s*`)
```

Comment:
this and the following changes seems unrelate, form where they come from?

Reply:
So, the updated yaml indent it differently during https://github.com/openshift-kni/performance-addon-operators/blob/c44791222457c701c7d067364bf1443e6377cfae/pkg/controller/performanceprofile/components/tuned/tuned_test.go#L48, so it looks like 
```
...
\nchannels=combined
      4\n
...
```

Reply:
lgtm but let's see if @yanirq agrees

Reply:
indenting is affected by the go version in this specific case ?

Reply:
nope, by the yaml version